model/commitqueue/commit_queue.go

```diff
@@ -187,21 +187,6 @@ func (q *CommitQueue) FindItem(issue string) int {
 	return -1
 }
 
-// EnsureCommitQueueExistsForProject inserts a skeleton commit queue if project doesn't have one
```

Comment:
It looks like this whole file can be deleted. Are we waiting to do this in a separate ticket/PR?

Reply:
Yep! It's not as simple as just deleting since some of the functions (like FindItem) are still used in places that also need to be deleted, so yeah just breaking it up